src/lock_commit/command_ext.rs

```diff
@@ -39,19 +31,8 @@ pub enum NetworkCommand {
         new_view: u128,
         highest_lock: CommandView,
     },
-}
-
-impl Command {
-    /// Send this command over to a server at the given address and return the response.
-    pub async fn send_to(self, address: SocketAddr) -> Result<Option<String>> {
-        let mut sender = ReliableSender::new();
-
-        let message: Bytes = bincode::serialize(&self)?.into();
-        let reply_handler = sender.send(address, message).await;
-
-        let response = reply_handler.await?;
-        let response: Result<Option<String>, String> = bincode::deserialize(&response)?;
-        response.map_err(|e| anyhow!(e))
+    Foward {
```

Comment:
not sure what the rest thinks but I would consider using the same naming method in blockchain and here, in the blockchain implementation the network messages are Message enums, and the client command is the Command variant of that Message enum. So instead of `Command::Forward{command: client_command}` you end up having `Message::Command(client_command)`

Reply:
I think conceptually I prefer having an enum that can have two variants (one for client messages and one for network messages), although I do prefer renaming it for consistency and also clarity (current implementation can be confusing in this implementation). No strong opinions though so I'm OK with whatever the people prefer.

Reply:
Yes for sure it's more important to be consistent across the project than the actual naming scheme we pick. I like the network vs client enum that you propose. Ideally I'd prefer if that could be worked into avoiding a special Forward variant within network commands though

Reply:
Btw these enum changes can be done in another PR